packages/interbit/src/args/getConfig.js

```diff
@@ -13,6 +14,11 @@ const getConfigLocation = () => {
   return configLocation
 }
 
+/**
+ * Gets the interbit configuration file from disk based on the --config
```

Comment:
We can use Asciidoc markup in descriptions, so mentions of command-line arguments, function names, parameter names, etc. should appear in monospace.

For this line, `--config` should be:

```
...based on the `--config`
```